pkg/medusa/reconcile.go

```diff
@@ -182,6 +189,100 @@ func UpdateMedusaMainContainer(dcConfig *cassandra.DatacenterConfig, medusaSpec
 	}
 }
 
+func medusaMainContainer(dcConfig *cassandra.DatacenterConfig, logger logr.Logger) (int, bool, *corev1.Container) {
```

Comment:
This function returns a lot of values, can we please use named returns to make it clearer + more self documenting?